src/resources/breaking-changes.md

```diff
@@ -0,0 +1,821 @@
+---
+title: Breaking changes and deprecations
+description: A list of breaking changes by release in Dart.
+---
+
+This page lists all language and library breaking changes and deprecations in Dart,
+organized by release and area, to help Dart users understand and manage their
+impact. Complete release notes are available in the [Dart SDK changelog][changelog].
+The [breaking change policy][] document describes the policy and process
+around breaking changes and deprecations in Dart. 
+
+**This page includes two types of breaking changes, as well as deprecations**:
+* **Unversioned**: The compiler does not maintain backward compatibility, and
```

Comment:
Was "compiler" explicitly chosen here? As far as I understand, breaking changes can result from tooling other than the compiler itself. Such as the analyzer introducing a new error, a tool being removed or changed (`dartfmt` in 2.14), etc.

Perhaps "Dart does not guarantee..."? Or the "(Dart) SDK does not guarantee..."

Reply:
Yeah I got it from [this suggestion](https://github.com/dart-lang/site-www/pull/5061#discussion_r1268755106) but I'm not tied to that phrasing, I'll change it to "Dart/SDK"